### DISCUSSION TITLE
import type `MenuPortalProps` error

### DISCUSSION THREAD

**Discussion by franco-aldeco** [JedWatson/react-select#5691] [Q&A]
how to import `MenuPortalProps` type in typescript?

**Comment by mp3846** [comment]
I just ran into this problem and simply modified `node_modules\react-select\dist\declarations\src\index.d.ts`:
```
export type { MenuListProps, MenuProps, NoticeProps, MenuPortalProps } from './components/Menu';
```

I don't know if it is missed or intentionally omitted
